logrotate/defaults.yaml

```diff
@@ -1,16 +1,15 @@
 # -*- coding: utf-8 -*-
 # vim: ft=yaml
-
+---
 logrotate:
   pkg: logrotate
   conf_file: '/etc/logrotate.conf'
   include_dir: '/etc/logrotate.d'
   user: root
   group: root
   service: cron
-  login_records_jobs: True
+  login_records_jobs: true
```

Comment:
YAML normally recognize 'True' as 'true', I don't think it needs to be changed.

Reply:
@daks Using the `default` configuration for `yamllint`, this gets flagged as needing to be changed, as shown in the error list in this PR's first comment.  Here's the relevant upstream documentation for it:

* https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy

We _can_ change this setting if necessary and we could even maintain different values per formula -- I've configured `ssf-formula` to handle having different `.yamllint` files per formula.  But is that a good precedent to set?  Keeping as close to the default configuration options seems like the best option, in my opinion -- leaves less room for confusion and more consistency in our formulas.  I've already come across a number of files which are using a mixture of `true`, `True`, `false` and `False` (not to mention `on` as well).

Reply:
I'm in favour of being consistent, it's just that True/False are Python-idioms so easier for Python developers :)
I'm OK with any choice.

Reply:
I understand, I've been using `True` and `False` exclusively up to now.  Maybe it's a good thing in it's own way -- it stops us thinking in Python and concentrate on the YAML instead!  A bit like getting used to `~` instead of `+` for Jinja string concatenation...

Reply:
Fine for me :)